src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                +
```

Comment:
could we a bit change current `toString` method by adding `+` operator at the beginning of each line? It better fits [this](https://mate-academy.github.io/style-guides/java/java.html#s4.5.1-line-wrapping-where-to-break) style point

Reply:
My first varient of resolving this task was absolutely the same as you said, but when I'm trying to pass the maven checkstyle tests it complains that I have to put '+' character on a new line below. So, what I have to do? Waiting for a 'Build Success' or follow the Google Ceckstyle?